src/Symfony/Component/Validator/Tests/Constraints/LengthValidatorTest.php

```diff
@@ -301,7 +308,7 @@ public function testInvalidValuesExactLessThanFourNamed($value)
     /**
      * @dataProvider getFiveOrMoreCharacters
      */
-    public function testInvalidValuesExactMoreThanFour($value)
+    public function testInvalidValuesExactMoreThanFour($value, $valueLength)
```

Comment:
```suggestion
    public function testInvalidValuesExactMoreThanFour(int|string $value,  int $valueLength)
```

same for the others

Reply:
I'm resolving this because adding parameter types to test cases doen't bring any. On the contrary, it might hide issues with type juggling.